osi_trafficsign.proto

```diff
@@ -9570,12 +9570,23 @@ message TrafficSign
         //
         VARIABILITY_OTHER = 1;
 
-        // Fixed sign, i.e. always present.
+        // Sign that can change neither semantically nor positionally, e.g.
+        // fixed immutable traffic sign.
         //
         VARIABILITY_FIXED = 2;
 
-        // Temporary or variable sign, e.g. on a sign bridge.
+        // Sign that can change semantically and positionally.
         //
         VARIABILITY_VARIABLE = 3;
+
+        // Sign that can change positionally but not semantically, e.g.
+        // temporary immutable traffic sign at construction site.
+        //
+        VARIABILITY_VARIABLE_MOVABLE = 4;
```

Comment:
Given the suggestion to add VARIABLE_MOVABLE_AND_MUTABLE, it is probably a good idea to drop the VARIABLE_ part for all of the new enum values:

```suggestion
        VARIABILITY_MOVABLE = 4;
```